### DISCUSSION TITLE
[HW4] Prob 3.1

### DISCUSSION THREAD

**Discussion by emjay73** [seominjoon/kaist-ai605#170] [Q&A] [answered]
**Problem 3.1** *(3 points)* Finetune your `bert-base-cased` model for `squad` question answering dataset, following a similar procedure to Problem 2.1. Report your accuracy here.

In problem 3.1, does accuracy mean f1 score and exact_match score? as in https://github.com/huggingface/transformers/tree/master/examples/pytorch/question-answering ??

**Comment by ssong18** [comment]
I personally just checked for the start and end token index, as they were the outputs from my model, and only marked the validation question correct if both were correct. (Not sure if this is what the professor meant by accuracy though.)

**Reply by emjay73** [reply]
That's what the exact_match tries to measure, while exact_match compared string to string, not index.
I expect exact_match and the way you measured to have similar values.
However, since exact_match performs sentence normalization, so I'm not that confident.

**Comment by emjay73** [comment]
I submitted with f1 score and exact_match score.
I'd like to be informed if it can cause a deduction in points.

**Comment by seominjoon** [comment, marked as answer]
Either F1 or EM is fine (EM is more often used these days though).
